lua/lspconfig/server_configurations/astro.lua

```diff
@@ -11,19 +11,13 @@ return {
     cmd = { 'astro-ls', '--stdio' },
     filetypes = { 'astro' },
     root_dir = util.root_pattern('package.json', 'tsconfig.json', 'jsconfig.json', '.git'),
-    init_options = {
```

Comment:
these lines should keep just remove tsdk

Reply:
Just `init_options` or should we also keep `init_options.typescript`?

Reply:
`init_options.typescript`.

Reply:
Please see if it's correct.

Reply:
lgtm